docs/docs/pyproject.md

```diff
@@ -123,29 +85,9 @@ To install a group of optional dependencies:
 $ pdm install -s socks
```

Comment:
Sorry, this is my first GitHub-base review, so I'm struggling a little bit with the process.

Anyway, above is still an example of `project.optional-dependencies` containing a `test` group which would now fit better into the `dev-dependencies`, right?

Reply:
Yes, just to wipe them out from the distribution metadata